### PR TITLE
removed default action from project.js #1716

### LINKED ISSUE #1716: Crispy Forms horizontal layout will be deactivated by project.js
I tried some days to use horizontal forms with crispy forms but it didn't work until I found that the default project.js contains the command to remove the row attribute from the form group:

```js
$('.form-group').removeClass('row');
```

Doesn't make it more sense to remove it?

### PR DESCRIPTION
**Pull request by hendrikschneider**
[//]: # (Thank you for helping us out: your efforts mean great deal to the project and the community as a whole!)

[//]: # (Before you proceed:)

[//]: # (1. Make sure to add yourself to `CONTRIBUTORS.rst` through this PR provided you're contributing here for the first time)
[//]: # (2. Don't forget to update the `docs/` presuming others would benefit from a concise description of whatever that you're proposing)

## Description
Closes #1716.

[//]: # (What's it you're proposing?)
Default should be a project.js that does not modify the html in some way

## Rationale

[//]: # (Why does the project need that?)

A cookiecutter project should generate a good starting point to start with and all modules should work without problems. This undocumented feature breaks crispy horizontal forms in a default setup and it is not obvious that there there is line in the project.js that removes css classes.

## Use case(s) / visualization(s)

[//]: # ("Better to see something once than to hear about it a thousand times.")

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/js/project.js [hunk 1]
```diff
@@ -18,4 +18,3 @@ Issues with the above approach:
 3. Really Unforgiving: No way to override this behavior
 4. Undocumented: No mention in the documentation, or it's too hard for me to find
 */
-$('.form-group').removeClass('row');
```
Comment (browniebroke) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/js/project.js]: The comments above this line related to crispy forms should be removed if this line is removed.

Comment (hendrikschneider) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/js/project.js]: done. Should I stash the changes or is it fine like this?

Comment (browniebroke) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/static/js/project.js]: No, don't worry about it, we can squash from the Github UI.

**Comment by webyneter** [comment]
LGTM.

**Comment by browniebroke** [comment]
Thanks for the fix!

**Pull request merged**